### PR TITLE
Move jumbo, and burst logic server side

### PR DESCRIPTION
**Pull request by sargun**
This starts the VPC Service side of work to move jumbo and
burst server side.

### CODE REVIEW

**Comment by sargun** [comment]
This doesn't have any of the client-side code yet. That's a WIP.

File: vpc/proto/vpc.proto [hunk 1]
```diff
@@ -126,6 +126,8 @@ message AssignIPRequestV3 {
     bool idempotent = 13;
     // Jumbo: I want "default routes" with an MTU of "Jumbo"
     bool jumbo = 14;
+    bool burst = 15;
+    uint64 bandwidth = 16;
```
Comment (solarkennedy) [inline vpc/proto/vpc.proto:130]: what do you think about encoding the units in the name?

File: vpc/proto/vpc.proto [hunk 2]
```diff
@@ -126,6 +126,8 @@ message AssignIPRequestV3 {
     bool idempotent = 13;
     // Jumbo: I want "default routes" with an MTU of "Jumbo"
     bool jumbo = 14;
+    bool burst = 15;
```
Comment (solarkennedy) [inline vpc/proto/vpc.proto:129]: how about `burstable_bandwidth` ? Or a docstring for that this does?

File: vpc/proto/vpc.proto [hunk 3]
```diff
@@ -148,6 +150,10 @@ message AssignIPResponseV3 {
         uint32 mtu = 2;
         Family family = 3;
     }
+    message Bandwidth {
+        uint64 bandwidth = 1;
+        uint64 burst = 2;
```
Comment (solarkennedy) [inline vpc/proto/vpc.proto:155]: Is this `bandwidth_limit` or `bandwidth_max` or something?

File: vpc/proto/vpc.proto (hunk 1 above)
Comment (sargun) [reply vpc/proto/vpc.proto:130]: This is actually a good question on what the unit actually is...

File: vpc/proto/vpc.proto (hunk 3 above)
Comment (sargun) [reply vpc/proto/vpc.proto:158]: What about "ceil"?

File: vpc/proto/vpc.proto (hunk 1 above)
Comment (rgulewich) [reply vpc/proto/vpc.proto:131]: Big +1 on including the units in the name.

**Review by rgulewich — APPROVED** [review]

**Comment by rgulewich** [comment]
LGTM, but I don't think I understand why we're moving it server-side.

**Comment by sargun** [comment]
1. centralized > decentralized at sufficient scale
2. For benoit A/B test

**Pull request merged**